testsuite/modern-testsuite/src/test/java/org/sonatype/nexus/testsuite/raw/RawHostedIT.java

```diff
@@ -36,6 +36,10 @@
 {
   public static final String HOSTED_REPO = "raw-test-hosted";
 
+  public static final String TEST_PATH = "alphabet.txt";
+
+  public static final String TEST_CONTENT = "alphabet.txt";
```

Comment:
Are two constants with the same string need here? Seems like TEST_PATH would be sufficient.